broker/src/main/java/io/camunda/zeebe/broker/system/configuration/ThreadsCfg.java

```diff
@@ -10,6 +10,7 @@
 public final class ThreadsCfg implements ConfigurationEntry {
   private int cpuThreadCount = 2;
   private int ioThreadCount = 2;
+  private boolean metricsEnabled = false;
```

Comment:
:x: I would favor to put that under experimental so we can remove that without worrying.

Reply:
Yeah, that's a good point. Not sure why I didn't do this from the beginning :+1: